python/friesian/example/wnd/recsys2021/wnd_preprocess_recsys.py

```diff
@@ -236,6 +232,9 @@ def transform_label(tbl):
                           ", but got " + args.cluster_mode)
 
     start = time()
+    args.input_train_folder = os.path.join(args.data_dir, 'train')
+    args.input_test_folder = os.path.join(args.data_dir, 'test')
+    args.output_folder = os.path.join(args.data_dir, 'preprocessed')
```

Comment:
remove args, just input_train_folder, change the places of args.input_train_folder instead